### DISCUSSION TITLE
大麻合法化？

### DISCUSSION THREAD

**Discussion by roma0323** [1101-GS4719-C/Python-Community-GS4719-C#1548] [Q&A]
我雖然自己沒有體驗過大麻，但確聽聞過朋友使用過的經驗，我也去查了一些大麻合法化的正反論述，以下是查到一些正反論點，不知道大家怎麼看

優點：

經濟效益：合法化可創造大量就業機會和稅收收入。
醫療用途：合法化可促進醫療大麻研究和應用，幫助患者緩解痛苦。
犯罪減少：合法化可減少非暴力大麻相關犯罪，釋放執法資源應對其他嚴重犯罪。
缺點：

健康風險：大麻使用可能導致心理和生理健康問題，特別是在未成年人和長期使用者中。
法律問題：合法化可能導致與聯邦法律衝突和規範上的挑戰。
青少年風險：合法化可能增加青少年使用風險，需要有效的年齡限制和預防措施。

**Comment by poweichen00** [comment]
大麻合法化的正反觀點在社會上一直存在著不同的意見，而這些觀點都反映了不同的利益和擔憂，各種社會、健康和經濟因素。對於大麻合法化的立場，人們的觀點可能因個人價值觀、經驗和觀察而有所不同。

**Comment by larry40305** [comment]
大麻合法化一定會帶來巨大的濫用，尤其在台灣青少年之間，即使有法規限制青少年也很難喝止。

**Comment by bestbanana295** [comment]
我覺得在目前台灣的社會菸商對於類似娛樂性用藥的影響力太大了導致其他比菸商傷力小的用具都不能進到台灣，簑以要等到大麻真的進來可能還˙要一段噓間。
